java/java.html

```diff
@@ -908,7 +908,7 @@ <h4 id="s5.2.5-non-constant-field-names">5.2.5 Non-constant field names</h4>
 <p>Non-constant field names (static or otherwise) are written
 in <a href="#s5.3-camel-case">lowerCamelCase</a>.</p>
 
-<p>These names are typically nouns or noun phrases.  For example,
+<p>Field names mustn't consist only of one letter. Typically, they are nouns or noun phrases. For example,
```

Comment:
IMHO it's better to write
```suggestion
<p>Names of fields mustn't consist only of one letter. Typically, they are nouns or noun phrases. For example,
```
Link with [info](https://englishsir.ru/grammar/prinadlezhnost-v-anglijskom-yazyke/#:~:text=%D0%9F%D1%80%D0%B8%D0%BD%D0%B0%D0%B4%D0%BB%D0%B5%D0%B6%D0%BD%D0%BE%D1%81%D1%82%D1%8C%20%D0%BD%D0%B5%D0%BE%D0%B4%D1%83%D1%88%D0%B5%D0%B2%D0%BB%D0%B5%D0%BD%D0%BD%D1%8B%D1%85%20%D0%BF%D1%80%D0%B5%D0%B4%D0%BC%D0%B5%D1%82%D0%BE%D0%B2,%D0%97%D0%BD%D0%B0%D1%87%D0%B5%D0%BD%D0%B8%D0%B5%20%D1%8D%D1%82%D0%BE%D0%B3%D0%BE%20%D1%81%D0%BB%D0%BE%D0%B2%D0%B0)

Reply:
@issugy I guess it doesn't matter each of these options is correct, plus you can see that on the 908th line `field names` too)

Reply:
i believe that it says here not about one actual field but about fields in general, so it looks a little bit strange for me, as well as 908th line. I'd rather change both of them)

`Non-constant field names` --> `Names of non-constant fields`